skins/cat17/templates/Donation_Comment.html.twig

```diff
@@ -0,0 +1,62 @@
+{% extends 'Base_Layout.html.twig' %}
+
+{% block page_identifier %}donation-comment{% endblock %}
+
+{% block main %}
+	<main class="comment">
+		<form id="comment-form" action="#" method="post" class="row">
+			<div class="container">
+				<div class="row">
+					<div class="content col-xs-12">
+						<h2 class="h1">Spendenkommentar</h2>
+						<fieldset class="col-xs-12 col-sm-6">
+							<legend class="legend">Erzahlen Sie uns, warum sie gespendt haben.</legend>
```

Comment:
Could you extract all the text to the content repo? We'll take care of the proper wording.

Reply:
Sure. First going to make it work though, then I will make it nicer